### PR TITLE
PR: Update to FA 5.6.3 and new "update_fa5" setup.py subcommand

### PR DESCRIPTION
**Pull request by darkvertex**
Hey hey, it's me again! :wave: 

This upgrades to *FontAwesome 5.5.0* but this is an extra special update... :sparkles: 

As I'm kind of repeating the same actions over and over when I contribute new FA updates, I explored the idea of creating a setup.py subcommand to do the hard work for me.

Now we can just run:

```
> python setup.py update_fa5 --fa-version 5.5.0
running update_fa5
Downloading from URL: https://github.com/FortAwesome/Font-Awesome/releases/download/5.5.0/fontawesome-free-5.5.0-web.zip
Dumping updated "brands" charmap: /home/alan/dev/qtawesome/qtawesome/fonts/fontawesome5-brands-webfont-charmap.json
Dumping updated "brands" font: /home/alan/dev/qtawesome/qtawesome/fonts/fontawesome5-brands-webfont.ttf
Dumping updated "solid" charmap: /home/alan/dev/qtawesome/qtawesome/fonts/fontawesome5-solid-webfont-charmap.json
Dumping updated "solid" font: /home/alan/dev/qtawesome/qtawesome/fonts/fontawesome5-solid-webfont.ttf
Dumping updated "regular" charmap: /home/alan/dev/qtawesome/qtawesome/fonts/fontawesome5-regular-webfont-charmap.json
Dumping updated "regular" font: /home/alan/dev/qtawesome/qtawesome/fonts/fontawesome5-regular-webfont.ttf
Patching new MD5 hashes in: /home/alan/dev/qtawesome/qtawesome/iconic_font.py
New "brands" hash is: b69de69a4ff8ca0abe96ec0b0c180c5b
New "solid" hash is: 48f54f63d7711d0912a9a10205538fc4
New "regular" hash is: b48c48ea8457846a5695b139c377d3d1
Dumping updated file: /home/alan/dev/qtawesome/qtawesome/iconic_font.py

Finished!
Please check the git diff to make sure everything went okay.
You should also edit README.md and qtawesome/docs/source/usage.rst to reflect the changes.
```
Handy! :tophat:

### CODE REVIEW

**Comment by darkvertex** [comment]
Looks like my use of `urllib2` in setup.py made it unfriendly to Python 3. I'll look into that.

**Comment by darkvertex** [comment]
All good now. :sweat_smile:

File: setup.py [hunk 1]
```diff
@@ -1,25 +1,180 @@
...
+VERSION_NS = {}
+with open(os.path.join(HERE, 'qtawesome', '_version.py')) as f:
+    exec(f.read(), {}, VERSION_NS)
 
-with io.open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
+with io.open(os.path.join(HERE, 'README.md'), encoding='utf-8') as f:
     LONG_DESCRIPTION = f.read()
 
+
+class UpdateFA5Command(distutils.cmd.Command):
```
Comment (ccordoba12) [inline setup.py]: Please move this class to a file called `setupbase.py` next to `setup.py`. That way it'll be easier to read our `setup.py`.

Thanks!

Comment (darkvertex) [reply setup.py]: Done! (Sorry it took a little while to address this. :santa:)

**Comment by darkvertex** [comment]
I moved the code to a separate file as per your request and reused my new subcommand to update to FA 5.6.3 which is the latest as of writing this comment.

**Comment by ccordoba12** [comment]
0.5.4 is out!

**Comment by darkvertex** [comment]
Awesome! Thanks for releasing promptly. :+1:

**Pull request merged**